documentation/topics/resources/code-interfaces.md

```diff
@@ -300,6 +300,64 @@ define :follow_artist do
 end
 ```
 
+### Default Options
+
+You can provide default options that will be merged with client-provided options using the `default_options` configuration. This is useful for setting common options that should be applied to all calls of a particular interface function.
+
+`default_options` accepts either:
+- A keyword list of static options
+- A zero-arity function that returns a keyword list of dynamic options
+
+#### Static Default Options
+
+```elixir
+code_interface do
+  define :get_user, action: :read, get_by: [:id],
+    default_options: [
+      load: [:profile, :posts],
+      authorize?: true
```

Comment:
authorization is typically true by default, so lets omit that from this example.